posts/2020-09-30-microprofile-language-support-liberty.adoc

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "MicroProfile Language Support for Open Liberty Applications"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/kathrynkodama
+author_github: https://github.com/kathrynkodama
+seo-title: MicroProfile Language Support for Open Liberty Applications - OpenLiberty.io
+seo-description: Leverage the new MicroProfile Language Server along with existing VS Code tools to develop your Eclipse MicroProfile applications with Open Liberty.
+blog_description: "Leverage the new MicroProfile Language Server along with existing VS Code tools to develop your Eclipse MicroProfile applications with Open Liberty."
+---
+= MicroProfile Language Support for Open Liberty Applications
+Kathryn Kodama https://github.com/kathrynkodama
+
+The new MicroProfile Language Server provides added support for developing your Eclipse MicroProfile applications with Open Liberty. Earlier this month, the first release of the Eclipse MicroProfile Language Server was published as a joint effort between IBM and Red Hat. The Tools for MicroProfile VS Code extension consumes this language server, providing feature-rich support for MicroProfile APIs and properties in VS Code.
```

Comment:
```suggestion
The new MicroProfile Language Server provides added support for developing your Eclipse MicroProfile applications with Open Liberty. Earlier this month, the first release of the Language Server for Eclipse MicroProfile (Eclipse LSP4MP), which is an open incubator project under the Eclipse Foundation, was published. The Tools for MicroProfile VS Code extension consumes this language server, providing feature-rich support for MicroProfile APIs and properties in VS Code.
```

Reply:
If we want to note the participants of the Eclipse project (e.g. IBM, RH, the MP community), we can still.  Then, we will note them under the context of the Eclipse project.

@kathrynkodama @lauracowen ^^^

Reply:
Updated with the wording you suggested.  I don't feel we need to note the participants, just that it is an open incubator project under the Eclipse Foundation is sufficient.